roq-demonstration/src/main/resources/testGCM.properties

```diff
@@ -2,12 +2,15 @@
 
 #SECTION 1 FREQUENCY
 period=3000
+# Defines the period after which the request to the GCM time out
+hcmTIMEOUT= 3000
```

Comment:
Also, the purpose of this configuration item is unclear from the comment and the name: it is in the GCM's config, so it must influence the GCM, and the comment confirms that, but then the variable is called hcmTimeout, not gcmTimeout. Please make this clearer.